src/components/Navbar/Navbar.js

```diff
@@ -1,11 +1,17 @@
 import { Link } from 'react-router-dom';
+import './Navbar.css';
+import imgUrl from '../../img/person-48.png';
 
 const Navbar = () => (
   <nav>
-    <div>
-      <Link to="/">Books</Link>
-      <Link to="/categories">Categories</Link>
+    <span className="bookstore-title">
+      Bookstore CMS
+    </span>
+    <div className="nav-links">
+      <Link to="/" className="books-link">Books</Link>
+      <Link to="/categories" className="categories-link">Categories</Link>
     </div>
+    <img className="profile-icon" src={imgUrl} alt="" />
```

Comment:
- [ ] For profile icons you can use this svg icon, simply download it and change the color 👍🏼

![person](https://user-images.githubusercontent.com/69011963/173165292-c3c99095-832e-4016-8a05-a2e5bb81f96b.svg)